routers/api/v1/repo/file.go

```diff
@@ -119,11 +126,14 @@ func GetRawFileOrLFS(ctx *context.APIContext) {
 		return
 	}
 
-	blob, lastModified := getBlobForEntry(ctx)
+	blob, entry, lastModified := getBlobForEntry(ctx)
 	if ctx.Written() {
 		return
 	}
 
+	ctx.RespHeader().Set(giteaObjectTypeHeader,
+		string(files_service.GetObjectTypeFromTreeEntry(entry)))
```

Comment:
```suggestion
	ctx.RespHeader().Set(giteaObjectTypeHeader, string(files_service.GetObjectTypeFromTreeEntry(entry)))
```

Reply:
make one line not to long ...

I personally thing i is more readable...

Reply:
Either is fine to me, just curious. :D